ocpp-1-6-api-adapter/src/main/kotlin/com/izivia/ocpp/adapter16/mapper/CommonMapper.kt

```diff
@@ -128,11 +128,7 @@ abstract class CommonMapper {
     @Named("convertEVSEType")
     fun convertEVSEType(evse: EVSEType?): Int =
         if (evse != null) {
-            if (evse.connectorId != null && evse.connectorId != evse.id) {
-                throw IllegalArgumentException("Argument evse can't have different id and connectorId : ${evse.id} != ${evse.connectorId}")
-            } else {
-                evse.id
-            }
+            evse.connectorId!!
```

Comment:
since the connector id is nullable, shouldn't we use the evse id if the connector id is null, rather than raising an exception ?
If the EVSE has "combined connectors" that could be the case IMO, no?

Reply:
Yes, it seemed wrong doing it, I fix this right away !